tests/format/angular/icu-expression/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,193 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic-plural-01.html format 1`] = `
+====================================options=====================================
+parsers: ["angular"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<span i18n>Updated {minutes, plural, =0 {just now} =1 {one minute ago} other {{{minutes}} minutes ago}}</span>
+
+=====================================output=====================================
+<span i18n
+  >Updated
+  {minutes, plural,
```

Comment:
Will moving the start of the ICU expression onto a new line cause issues due to the extra whitespace being inserted?

Reply:
Prettier already formats the same way for interpolation. I thought it was the same as this, but is it not? Or this behaviour is a bug?

**Prettier 3.1.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAzgBwIZQHwCqmAJtvMQATDAUA2EDjTzLrb7H0A5hQL6+oA9Flx4QAGhARMMAJbR0yUNgBOKiAHcACqoSKU2WhuwBPRZIBGK7GADWcGAGVsAWzgAZWVDjIAZofQ4S2s7B0ccMC8uZBgVAFcgkECXWRj4xLgAD0w4FVk3WEMAFVyoVVk4fX9aQMl0KNo4AEU4iHg-AMSAK3RMxwbm1vakatqQAEchuC11TH0QbHQAWm84YjWJEFjsWVoogGEIFxdsZAXaWk36qC5GgEEYWNkLOPgtXM9vDprEgAsYFy0ADqv1k8BEYDgjj0YNkADcwSYzmB0OYQHCEgBJKDrWCOMB5GR3HGOGAmRrfMaYdSBIHWTBnamVXJwnySLyBFQwGbYLgnSmJHAqTlnXBcOK0VSbaleGBA2TEGC-ZAADgADJIVHBJrItTy+acRp1JDBsBZ5YrlUgAEySOKBIpmqrGkBwFwWNbrYjuMVxXlwABiEBUJ0eUVFrwgIH4QA)
<!-- prettier-ignore -->
```sh
--parser angular
```

**Input:**
<!-- prettier-ignore -->
```html
<span>Updated {{ loooooooooooooooooooooooooooooooooooooooooooooooooooooooong }}</span>
```

**Output:**
<!-- prettier-ignore -->
```html
<span
  >Updated
  {{ loooooooooooooooooooooooooooooooooooooooooooooooooooooooong }}</span
>

```

Reply:
Good point - I think it will be OK then like this 😄